src/store/index.js

```diff
@@ -132,22 +115,33 @@ export default createStore({
     },
     ADD_TO_CARD(state, payload) {
       const { currentUser, users, textInfo, products, shoppingCart } = state;
-      if (shoppingCart[0].full === false) {
-        shoppingCart[0].full = true;
-        shoppingCart[0].products[0].name = payload[0];
-        shoppingCart[0].products[0].points = payload[1];
-      } else {
-        state.cartInfo = "You have already added an item!";
-        setTimeout(() => {
-          state.cartInfo = "You can add only one item.";
-        }, 5000);
-      }
+      shoppingCart[0].full = true;
+      shoppingCart[0].products[shoppingCart[0].productsInCart].name =
+        payload[0];
+      shoppingCart[0].products[shoppingCart[0].productsInCart].points =
+        payload[1];
+      shoppingCart[0].productsInCart += 1;
+      shoppingCart[0].totalPrice += payload[1];
```

Comment:
Będzie to trzeba będzie przemianować na końcu, żeby można było dodawać nieskończenie wiele produktów